src/cli-app/pom.xml

```diff
@@ -174,6 +175,7 @@
                 </execution>
                 <execution>
                     <id>assemble-zip</id>
+                    <phase>package</phase>
```

Comment:
I will merge for now, and you can work combining the two assemblies correctly as a 2nd pull request.